piIOComm.h

```diff
@@ -32,6 +32,7 @@
 
 #pragma once
 
+#include <piControl.h>
```

Comment:
Hm, why is the additional include file needed?

Reply:
It is not, I will remove it.